### ISSUE TITLE
Could you please release the model pre-train weights?

### ISSUE THREAD

**Issue by Weepingchestnut** [hanlinwu/SADN#3]
We have read your paper 'Scale-Aware Dynamic Network for Continuous-Scale Super-Resolution'. It is very interesting.

We want to compare with your method and cite it in our work. 

Could you please release your model pre-train weight in order for a fair comparison?

**Comment by Morayabb** [comment]
Hello, may I ask if you can successfully run this code and if the loss value can be displayed correctly during the training process. Could you reply in your free time. Thank you!!

**Comment by hanlinwu** [comment]
I have uploaded the pretrained weights for the DIV2K dataset and the remote sensing dataset, please see [readme.md](https://github.com/hanlinwu/SADN/blob/master/readme.md).
